src/Features/Core/Portable/Completion/Providers/Snippets/SnippetData.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers.Snippets
+{
+    internal struct SnippetData
```

Comment:
What's the benefit of this type?

Reply:
A snippet has a few factors in its creation, the text change, cursor position, etc. SnippetData was created as a means to get all the available snippets at a particular location without needing to fill out all that information.